components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -44,6 +44,12 @@ public function __construct() {
 		// add the same tab for taxonomies
 		add_filter( 'pods_admin_setup_edit_tabs_taxonomy', array( $this, 'tab' ), 11, 3 );
 
+		// add the same tab for comments
+		add_filter( 'pods_admin_setup_edit_tabs_comment', array( $this, 'tab' ), 11, 3 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 0e8ebdc882d287c5f98071c62b8dfdd1</sub>